slc/component/matter-platform/display/matter_default_lcd_config.slcc

```diff
@@ -9,6 +9,9 @@ metadata:
     license: Zlib
 # Boards that support LCD with Matter projects
 requires:
+  - name: matter_lcd
```

Comment:
This file has been reworked in 2.7. I guess we will deal with the conflict at merge time.

Reply:
I know, it was a hack to get the LCD working until we get the merge completed. There might be more than one but since we can't merge this PR until we update main, I will be the one dealing with those in due time...